src/components/ActionButton/__tests__/ActionButton.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import ActionButton from '../ActionButton';
+
+const buttonText = 'Accept';
+const mockCallback = jest.fn();
+
+describe('<ActionButton />', () => {
```

Comment:
Does this only test the large action button because it has text and is therefore easier to get the element? Or was only one tested because they are so similar?

Reply:
because it had text and was easier to get.  I considered adding more tests for the different permutations, but decided to focus on adding more test elsewhere.